dts/arm/st/u5/stm32u595.dtsi

```diff
@@ -101,7 +101,7 @@
 		};
 
 		usbotg_hs: otghs@42040000 {
-			compatible = "st,stm32-otghs";
+			compatible = "st,stm32u5-otghs", "st,stm32-otghs";
```

Comment:
The `physel` propoerty seems required. Should this DT node (and few others in this series where only the compatible property is updated) also set a default `physel` property, possibly overridden by board DTS files?

Reply:
Property should indeed be required, but adding default is superfluous if we're populating the property all across the tree (specifically for cases where only one value is possible).
(Also `default` makes `required` less pertinent)